app/controllers/concerns/error_rescuable.rb

```diff
@@ -14,7 +14,7 @@ def user_not_authorized(exception)
     policy_name = exception.policy.class.to_s.underscore
     error = I18n.t "#{policy_name}.#{exception.query}", scope: 'pundit', default: :default
 
-    render json: { message: error }, status: :unprocessable_entity
+    render json: { message: error }, status: :unauthorized
```

Comment:
I think your branch is out of date, this line was already corrected :D. Can you please update it?

https://github.com/comarev/comarev/blob/master/app/controllers/concerns/error_rescuable.rb#L17

Reply:
You're right! my bad!